test/exe/Main.hs

```diff
@@ -110,8 +110,6 @@ initializeResponseTests = withResource acquire release tests where
                               , _willSaveWaitUntil = Nothing
                               , _save = Just (SaveOptions {_includeText = Nothing})}))
 
-      nothingWorkspace = Just (WorkspaceOptions {_workspaceFolders = Nothing})
```

Comment:
Can we get a test for this?

Reply:
I can add a test that no `didChangeWatchedFile` subscriptions are made for files outside the workspace. Is that what you had in mind?

Reply:
I was thinking of something similar to the existing test for `DidFileExists` that you added but with the file being outside of the workspace. Probably needs a direct-style cradle in a `hie.yaml` that has a `-i` flag pointing outside of the workspace.